src/modules/webext/chromium/shared/chromium-bookmark/chromium-bookmark.service.ts

```diff
@@ -13,73 +18,23 @@ import {
 import * as Exceptions from '../../../../shared/exception/exception';
 import Globals from '../../../../shared/global-shared.constants';
 import { WebpageMetadata } from '../../../../shared/global-shared.interface';
+import { NativeContainersInfo } from '../../../shared/webext-bookmark/NativeContainersInfo';
 import WebExtBookmarkService from '../../../shared/webext-bookmark/webext-bookmark.service';
 
 @autobind
 @Injectable('BookmarkService')
 export default class ChromiumBookmarkService extends WebExtBookmarkService {
-  otherBookmarksNodeId = '2';
-  toolbarBookmarksNodeId = '1';
+  otherBookmarksNodeTitle = 'Other bookmarks';
+  toolbarBookmarksNodeTitle = 'Bookmarks bar';
+  menuBookmarksNodeTitle = 'Menu bookmarks';
+  mobileBookmarksNodeTitle = 'Mobile bookmarks';
```

Comment:
This will break on any non-english locale. You have to use the id to detect the container. Note that even this is not perfect (as ids can change under some circumstances) and I have discussed with the chromium team - there is no API or "supported" way of retrieving containers.

Reply:
Got it! What about about letting the user specify the right folder during the setup?

Reply:
Ids work 99% of the time. Implementing a UI just to select these folders is a pretty large change for little gain.

Reply:
Ye... I guess implementing opera/kiwi specific version that just have different getNativeContainersIds would be the best approach then. It seems that Kiwi mobile bookmarks have id 3 (following chromiums id 1 and 2). Not sure what to do about opera, I read here something about the IDs not being static?